### PR TITLE
Removing futile flags from fall-forward or fall-back commands

### PR DESCRIPTION
**Pull request by priyanshi-yb**
fixes - 
1. https://yugabyte.atlassian.net/browse/DB-8524
2. https://yugabyte.atlassian.net/browse/DB-8341
3. https://yugabyte.atlassian.net/browse/DB-8523

### CODE REVIEW

File: yb-voyager/cmd/common.go [hunk 1]
```diff
@@ -578,3 +578,14 @@ func GetSourceDBTypeFromMSR() string {
 	}
 	return msr.SourceDBConf.DBType
 }
+
+func hideFlagsInFallForwardOrBackCmds(cmd *cobra.Command) {
+	var flags = []string{"target-db-type", "import-type", "source-db-type", "export-type", "target-endpoints", "use-public-ip", "continue-on-error", "table-list",
```
Comment (makalaaneesh) [inline yb-voyager/cmd/common.go:583]: fall-forward setup command can have table-list, no? Don't see a problem with that..

File: yb-voyager/cmd/fallBackSetup.go [hunk 2]
```diff
@@ -45,8 +45,9 @@ func init() {
 	registerCommonGlobalFlags(fallBackSetupCmd)
 	registerCommonImportFlags(fallBackSetupCmd)
 	registerSourceDBAsTargetConnFlags(fallBackSetupCmd)
+	registerImportDataCommonFlags(fallBackSetupCmd)
```
Comment (makalaaneesh) [inline yb-voyager/cmd/fallBackSetup.go:48]: thanks for this :) somehow this got slipped.

File: yb-voyager/cmd/common.go (hunk 1 above)
Comment (makalaaneesh) [reply yb-voyager/cmd/common.go:583]: pls ensure table-list flags are not hidden from fall-back/forward synchronize

Comment (priyanshi-yb) [reply yb-voyager/cmd/common.go:583]: Yes, we use different function for synchronize i.e. Export... one, so not removing from there.

Comment (priyanshi-yb) [reply yb-voyager/cmd/common.go]: yes done.

**Review by makalaaneesh — APPROVED** [review]
LGTM!

**Pull request merged**